osu.Game/Skinning/ISkinnableDrawable.cs

```diff
@@ -14,5 +14,11 @@ public interface ISkinnableDrawable : IDrawable
         /// Whether this component should be editable by an end user.
         /// </summary>
         bool IsEditable => true;
+
+        /// <summary>
+        /// <see langword="false"/> if this <see cref="ISkinnableDrawable"/>'s <see cref="Drawable.Anchor"/> is automatically determined by proximity,
+        /// <see langword="true"/> if the user has overridden it.
+        /// </summary>
+        bool OverridesClosestAnchor { get; set; }
```

Comment:
I think you could get away with default interface implementation instead, that would work amicably without having to implement this everywhere:
```suggestion
        public bool OverridesClosestAnchor { get; set; }
```

Reply:
Also not really sure about the change from `UsingClosestAnchor` to `OverridesClosestAnchor`, the old way reads better IMO. (could even rename to `UsesClosestAnchor`)

Reply:
Do you mean the C# 8 feature? I tried that but I don't believe those can hold any kind of state (auto properties create implicit fields).

Reply:
Unresolving this because I don't like `OverridesClosestAnchor` either. `UsesClosestAnchor` seems better. Or if you want to go the converse route, `UsesFixedAnchor` as a negation of that.